tests/unit_tests/graph_test.py

```diff
@@ -54,7 +55,7 @@ def test_enrichment_tree_search(generate_proteins):
         "cytoplasm",
     ]
     result_orig = enrichment.test_enrichment(df, go_subset=terms)
-    result_tree_search = enrichment.test_enrichment(
+    result_graph_search = enrichment.test_enrichment(
         df, go_subset=terms, aggregate_terms=True
     )
-    assert result_orig.equals(result_tree_search)
```

Comment:
the doctoring says it should give diff results, but it seems like the opposite is being tested here

Reply:
nice!